flank-scripts/src/main/kotlin/flank/scripts/pullrequest/SetLabels.kt

```diff
@@ -0,0 +1,44 @@
+package flank.scripts.pullrequest
+
+import com.github.kittinunf.fuel.Fuel
+import com.github.kittinunf.fuel.coroutines.awaitResult
+import com.github.kittinunf.fuel.coroutines.awaitStringResult
+import com.github.kittinunf.result.getOrNull
+import com.github.kittinunf.result.map
+import com.github.kittinunf.result.onError
+import com.github.kittinunf.result.success
+import flank.scripts.exceptions.mapClientError
+import flank.scripts.exceptions.toGithubException
+import flank.scripts.github.GitHubLabelDeserializable
+import flank.scripts.utils.toJson
+import kotlinx.serialization.Serializable
+
+suspend fun copyLabels(githubToken: String, issueNumber: Int, pullRequestNumber: Int) {
+    getLabelsFromIssue(githubToken, issueNumber)
+        .onError { println("Could not copy labels because of ${it.message}") }
+        .map { it.map { label -> label.name } }
+        .getOrNull()
+        ?.run { setLabelsToPullRequest(githubToken, pullRequestNumber, this) }
+}
+
+private suspend fun getLabelsFromIssue(githubToken: String, issueNumber: Int) =
+    Fuel.get("https://api.github.com/repos/Flank/flank/issues/$issueNumber/labels")
+        .appendHeader("Accept", "application/vnd.github.v3+json")
+        .appendHeader("Authorization", "token $githubToken")
```

Comment:
In ```GitHubApi.kt``` we have function ```appendHeaders```. This function adds these headers to the request. Maybe we could use these functions in this and other places? WDYT?

Reply:
I think that this is very good idea